src/MCP2515.cpp

```diff
@@ -288,6 +290,10 @@ int MCP2515Class::filter(int id, int mask)
 
   for (int n = 0; n < 2; n++) {
     // standard only
+    // TODO: This doesn't look correct. According to the datasheet, the RXM0 and
+    // RMX1 should either both be unset (in which case filters are active), or
+    // both be unset (in which case all filters are ignored).
```

Comment:
This appears to be a hangover from the MCP2510 which is no longer (officially) supported by the MCP2515. See https://i.imgur.com/QmthNtj.png vs https://i.imgur.com/sGE2jpq.png

Reply:
Thanks for finding this!
I suspect it's just a typo and the intention was to use `FLAG_RXM1` on L292 (old).
But at the same time I don't want to change code that I don't fully understand, and didn't spend enough time testing. The PR as-is is already an improvement, and hopefully with the TODO someone else (or future me) will be curious to figure out how to fix it.